<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceLamp</class>
 <widget class="QWidget" name="DeviceLamp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_background">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_name_lamp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>汉仪旗黑-45S</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>吊灯</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_lamp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>175</width>
        <height>255</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 171, 171);</string>
      </property>
      <property name="text">
       <string>灯图片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_onoff">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
